addons/website_sale/views/templates.xml

```diff
@@ -264,36 +265,96 @@
             <t t-set="additional_title">Shop</t>
             <t t-set="grid_block_name">Grid</t>
             <t t-set="product_block_name">Product</t>
+
+            <!-- Qweb variable defining the class suffix for navabar items.
+                 Change accordingly to the derired visual reault (eg. `primary`, `dark`...)-->
+            <t t-set="navClass" t-valuef="light"/>
+
+            <!-- Check for active options: the stored value may be used in sub-templates too  -->
+            <t t-set="opt_wsale_categories" t-value="is_view_active('website_sale.products_categories')"/>
+            <t t-set="opt_wsale_attributes" t-value="is_view_active('website_sale.products_attributes')"/>
+            <t t-set="opt_wsale_filter_price" t-value="is_view_active('website_sale.filter_products_price')"/>
+
+            <t t-set="opt_wsale_categories_top" t-value="is_view_active('website_sale.products_categories_top')"/>
+            <t t-set="opt_wsale_attributes_top" t-value="is_view_active('website_sale.products_attributes_top')"/>
+
+            <t t-set="website_sale_pricelists" t-value="website.get_pricelist_available(show_visible=True)" />
+            <t t-set="website_sale_sortable" t-value="website._get_product_sort_mapping()"/>
+
+            <t t-set="hasLeftColumn" t-value="opt_wsale_categories or opt_wsale_attributes"/>
+
+            <t t-set="isFilteringByPrice" t-if="opt_wsale_filter_price" t-value="available_min_price != min_price or available_max_price != max_price"/>
+            <t t-set="hasPricelistDropdown" t-value="website_sale_pricelists and len(website_sale_pricelists)&gt;1"/>
+            <t t-set="isSortingBy" t-value="[sort for sort in website_sale_sortable if sort[0]==request.params.get('order', '')]"/>
+
             <div id="wrap" class="js_sale o_wsale_products_page">
                 <div class="oe_structure oe_empty oe_structure_not_nearest" id="oe_structure_website_sale_products_1"/>
                 <div class="container oe_website_sale pt-2">
-                    <div class="row o_wsale_products_main_row">
-                        <div t-if="enable_left_column" id="products_grid_before" class="col-lg-3 pb-2">
-                            <div class="products_categories"/>
-                            <div class="products_attributes_filters"/>
-                        </div>
-                        <div id="products_grid" t-attf-class="col #{'o_wsale_layout_list' if layout_mode == 'list' else ''}">
+                    <div class="row o_wsale_products_main_row align-items-start flex-nowrap">`
```

Comment:
I believe the backtick is not wanted here
```suggestion
                    <div class="row o_wsale_products_main_row align-items-start flex-nowrap">
```

Reply:
ooops, thanks :/